go.mod

```diff
@@ -3,12 +3,15 @@ module github.com/optimizely/go-sdk
 go 1.12
 
 require (
+	github.com/DATA-DOG/godog v0.7.13
```

Comment:
Hmm we need to find a way to ignore these test dependencies when we `go get` 

These are only used for the cucumber step definitions and we are planning on just running in docker to not pollute this go.mod file. I would rather not have to declare test deps in here.

For now the best thing to do is just revert these changes until we figure out a better way to handle test dependencies.